### DISCUSSION TITLE
Error compiling Slint for ESP-IDF on Windows

### DISCUSSION THREAD

**Discussion by isthaison** [slint-ui/slint#4026] [Ideas]
After build esp ->  edit file build.nijia

![image](https://github.com/slint-ui/slint/assets/22739578/b2b9f0b4-6218-4a90-ba54-c394052b4dda)

old
![image](https://github.com/slint-ui/slint/assets/22739578/8ba9ad8c-a2d2-4212-b8d5-d01f113cd8ee)

after changes
![image](https://github.com/slint-ui/slint/assets/22739578/428cec3a-dd7f-45ec-bc51-ae443c291d0b)

It seems I understand windows won't understand the path when minifying .exe

**Comment by isthaison** [comment]
![image](https://github.com/slint-ui/slint/assets/22739578/8b8bfa58-c25f-42c9-9a43-45f61f76291d)
It has working

**Reply by tronical** [reply]
Wohoo :)

**Comment by tronical** [comment]
The build issue you're running into was fixed in https://github.com/corrosion-rs/corrosion/pull/437 . This will be included in the next Corrosion release, and consequently in the next Slint release after that. In the meantime, a workaround is to use WSL.

**Reply by isthaison** [reply]
Something, do you know reason touch, UI pixel then error ![image](https://github.com/slint-ui/slint/assets/22739578/de236677-71fb-49c6-bfd6-32a3ad705d78)
